File: web/src/main/java/no/nav/modiapersonoversikt/service/utbetaling/UtbetalingService.java
package no.nav.modiapersonoversikt.service.utbetaling;

import no.nav.tjeneste.virksomhet.utbetaling.v1.informasjon.WSUtbetaling;
import org.joda.time.LocalDate;

import java.util.List;

public interface UtbetalingService {
    List<WSUtbetaling> hentWSUtbetalinger(String fnr, LocalDate startDato, LocalDate sluttDato);

    void ping();
}

File: web/src/main/java/no/nav/modiapersonoversikt/service/utbetaling/UtbetalingService.kt
package no.nav.modiapersonoversikt.service.utbetaling

import no.nav.common.types.identer.Fnr
import no.nav.modiapersonoversikt.infrastructure.ping.Pingable
import java.time.LocalDate

interface UtbetalingService : Pingable {
    fun hentUtbetalinger(
        fnr: Fnr,
        startDato: LocalDate,
        sluttDato: LocalDate,
    ): List<UtbetalingDomain.Utbetaling>
}
